### ISSUE TITLE
Checkbox: lack of accessibility via keyboard

### ISSUE THREAD

**Issue by zaaakher** [sikka-software/Hawa#85]
### Description

When the focus is in the checkbox, hitting enter doesn't it check it

**Comment by zaaakher** [comment]
Currently we can check and uncheck using the spacebar. Which seems to be the default normal behavior
